vbr/vbrCore/functions/sr_visc_calc_BKHK2023.m

```diff
@@ -0,0 +1,77 @@
+function VBR= sr_visc_calc_BKHK2023(VBR)
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%
+% VBR= sr_visc_calc_BKHK2023(VBR)
+% DH 9/26/2025
```

Comment:
totally OK with me if you want to leave your initials here :) 

but your contribution will also be recorded in the git history and the release notes (https://vbr-calc.github.io/vbr/history/)